src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/GeneralApplicationDirectionsAboutToSubmitHandlerTest.java

```diff
@@ -82,7 +85,7 @@ public class GeneralApplicationDirectionsAboutToSubmitHandlerTest extends BaseHa
     public void setup() {
         objectMapper = new ObjectMapper();
         startHandler = new GeneralApplicationDirectionsAboutToStartHandler(
-            assignCaseAccessService, finremCaseDetailsMapper, helper, service);
+            assignCaseAccessService, finremCaseDetailsMapper, helper, service, partyService);
         submitHandler = new GeneralApplicationDirectionsAboutToSubmitHandler(
             finremCaseDetailsMapper, helper, service, gaService, generalApplicationsCategoriser);
     }
```

Comment:
Refactor this test to @ExtendWith(MockitoExtension.class). Take HearingsAboutToStartHandlerTest.java as an example.